### PR TITLE
Python 3.6 Compatibility

### PR DESCRIPTION
**Pull request by Coresummer**
Instead of using str.removesuffix, changed to using re.sub so the program works even below python3.9

### CODE REVIEW

**Comment by riita10069** [comment]
Welcome home,  @Coresummer !!

/assign @riita10069 
/cc @yurimarimo 


Thanks for your great contribution:))
I would like to know more about the motivation behind this PR.
I have heard from you that you want it to work with older versions of Python. That's great. I feel the same way.
So, when I incorporate your modifications, will it work as old as Python 3.x. What's the number that goes in the x?
Can you actually run it on your hand to check?
I don't have any tests, so it's a hassle.lol

File: mdt.py [hunk 1]
```diff
@@ -18,13 +18,14 @@ def get_dest_file_path(filename, src_lang, dest_lang, output):
     current_directory = Path.cwd()
 
     if output_directory == current_directory:
-        dest_file_path = Path.resolve(Path('.' + filename + '.md' if dest_lang == const.LANG_EN else filename + '.' + dest_lang + '.md'))
+        dest_file_path = Path.resolve(Path(filename + '.md' if dest_lang == const.LANG_EN else filename + '.' + dest_lang + '.md'))
```
Comment (riita10069) [inline mdt.py]:
```suggestion
        dest_file_path = Path.resolve(Path(filename + '.md' if dest_lang == const.LANG_EN else filename + '.' + dest_lang + '.md'))
```

File: mdt.py [hunk 2]
```diff
@@ -96,29 +97,26 @@ def run(path, recursive, src_lang, dest_lang, output, debug):
         if (path.endswith("." + src_lang + ".md") or (
                     src_lang == const.LANG_EN and "." not in path.split("/")[-1].rstrip(".md"))):
             translate_page(
-                path.removesuffix('.md') if src_lang == const.LANG_EN else path.removesuffix('.' + src_lang + '.md'),
+                re.sub('\.md$', '', path) if src_lang == const.LANG_EN else re.sub('.' + src_lang + '.md', '', path),
```
Comment (yurimarimo) [inline mdt.py]: ここ、エスケープを忘れてるように見える！
```suggestion
                re.sub('\.md$', '', path) if src_lang == const.LANG_EN else re.sub('\.' + src_lang + '.md', '', path),
```

Comment (riita10069) [reply mdt.py]: 天才！！全然気づかなかった。

**Review by riita10069 — APPROVED** [review]

**Comment by Coresummer** [comment]
Personally checked python3.6 works but python3.5. So at least this lib should work from py3.6~

**Pull request merged**